.eslintignore

```diff
@@ -1,3 +1,4 @@
 build/*
 **/coverage*
 packages/*/dist
+packages/core-utils/utils
```

Comment:
why is this added?

Reply:
This is because of this change: https://github.com/Synthetixio/synthetix-v3/pull/1055/files#diff-c213856001d7550e3bd55730a1a2b2dcb919c90e11946a3b500f10bf4557386bR6

Which makes the build to be on the root instead of the `dist` folder, which allows us to have a clearer import like `import '@synthetixio/core-utils/utils/*'`. It's a common pattern used also by hardhat eg. [here](https://github.com/NomicFoundation/hardhat/tree/master/packages/hardhat-core/src)